### PR TITLE
Correct broken link typo in Containerized AAP Installation Guide

### PR DESCRIPTION
**Pull request by michellemacrh**
Correct broken link to "Setting up an unbound DNS server - Red Hat Enterprise Linux 9"

[DDF] Link is broken

https://issues.redhat.com/browse/AAP-16502

### CODE REVIEW

File: downstream/modules/platform/ref-aap-containerized-dns-config.adoc [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
 
-Optional: To have the installer automatically pick up and apply your AAP subscription manifest license, then follow this guide up to and including step 11, Tp generate a manifest file which can be downloaded for the installer refer to link:https://docs.ansible.com/ansible-tower/latest/html/userguide/import_license.html#obtain-sub-manifest[4. Import a Subscription — Ansible Tower User Guide v3.8.6].
+Optional: To ensure the installation program automatically picks up and applies your {PlatformNameShort} subscription manifest license, generate a manifest file. You can then download this manifest file for the installation program to refer to. To generate the manifest file, see link:https://docs.ansible.com/ansible-tower/latest/html/userguide/import_license.html#obtain-sub-manifest[4.1. Obtaining a subscriptions manifest - Ansible Tower User Guide v3.8.6].
```
Comment (ianf77) [inline downstream/modules/platform/ref-aap-containerized-dns-config.adoc:17]: You may have to update this link before the end of the year. The Controller User Guide contains a section of creating a Subscription Manifest.

**Review by ianf77 — COMMENTED** [review]
One comment

File: downstream/modules/platform/ref-aap-containerized-dns-config.adoc (hunk 1 above)
Comment (michellemacrh) [reply downstream/modules/platform/ref-aap-containerized-dns-config.adoc:17]: Ah good to know. I can make a note to update this again when the controller docs are live if that is suitable? 
Is there an ETA on the Controller User Guide? I know yourself and Ed are working away on it.

Comment (michellemacrh) [reply downstream/modules/platform/ref-aap-containerized-dns-config.adoc:17]: I can make a note to update this link again when the Controller User Guide is published if that's suitable?

I can update the link for now with that updated link you provided, thank you :)

Comment (ianf77) [reply downstream/modules/platform/ref-aap-containerized-dns-config.adoc:17]: Maybe end of November if we're lucky. However the published Getting Started guide still has links to the Controller User docs, and they all have to be updated once it's been moved to Customer Portal

Comment (michellemacrh) [reply downstream/modules/platform/ref-aap-containerized-dns-config.adoc]: Good to know. Is there a Jira ticket or a spreadsheet to track updating references and that type of work for the Controller docs?

Comment (ianf77) [reply downstream/modules/platform/ref-aap-containerized-dns-config.adoc]: Not yet. It's something I have planned. But as were only just over half way through. I was going to wait until later on. You could create one and link it to the Epic AAP-15342.

Comment (michellemacrh) [reply downstream/modules/platform/ref-aap-containerized-dns-config.adoc]: No problem, I can do that so there's a place to track all this type of info. You can cross it off your list! :)

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**